### DISCUSSION TITLE
How to make node details follow the same color scheme as the node?

### DISCUSSION THREAD

**Discussion by Foxxsayy** [freeplane/freeplane#1337] [Help]
I'd like to make the details section have a different text characteristics, but have the same color for the background, outline, text color, etc. (Or a slightly lighter/darker shade than the node). I'm trying to figure out how to do this with the rules, but I'm having some trouble with it.

**Comment by macmarrum** [comment]
You can have a look at the wiki docs:
https://docs.freeplane.org/user-documentation/styles.html
